File: Solo Learn/Fácil/Varandas/balconies.cpp
#include <stdio.h>
#include <stdlib.h>

int main() {
    // Variables to store dimensions and areas
    int heightA, widthA, heightB, widthB, areaA, areaB;

    // Input dimensions for apartment A
    scanf("%d,%d", &heightA, &widthA);

    // Input dimensions for apartment B
    scanf("%d,%d", &heightB, &widthB);

    // Calculate areas
    areaA = heightA * widthA;
    areaB = heightB * widthB;

    // Compare areas and output result
    if (areaA > areaB) {
        printf("Apartment A\n");
    } else if (areaB > areaA) {
        printf("Apartment B\n");
    } else {
        printf("Both apartments have balconies of the same size.\n");
    }

    return 0;
}

File: Solo Learn/Fácil/Varandas/balconies.py
# Função principal
def main():
    # Entrada de dimensões para o apartamento A
    dimensionsA = input().split(',')
    heightA, widthA = map(int, dimensionsA)

    # Entrada de dimensões para o apartamento B
    dimensionsB = input().split(',')
    heightB, widthB = map(int, dimensionsB)

    # Calculando as áreas
    areaA = heightA * widthA
    areaB = heightB * widthB

    # Comparando as áreas e exibindo o resultado
    if areaA > areaB:
        print("Apartment A")
    elif areaB > areaA:
        print("Apartment B")
    else:
        print("Both apartments have balconies of the same size.")

# Executa o programa
if __name__ == "__main__":
    main()